jukebox/src/main/java/com/spsteam14/jukebox/DataServlet.java

```diff
@@ -0,0 +1,44 @@
+package com.spsteam14.jukebox;
+
+import com.google.appengine.api.utils.SystemProperty;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.cloud.firestore.FirestoreException;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.cloud.firestore.WriteResult;
+import com.google.api.core.ApiFuture;
+import com.google.gson.Gson;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.CollectionReference;
+import java.lang.Math;
+import java.util.*;
+import com.google.cloud.firestore.EventListener;
+import javax.annotation.Nullable;
+
+/** Ignore this servlet for now; backend will change from Java to another language eventually*/
```

Comment:
I'm a bit unsure about this file - if it's not being used for anything, do we need to check it in?

Reply:
I kept it just in case we tried to go back and get things working on the servlet, but since Jeremy and I have both been working with Javascript, I think you're right -- it's a good call to get rid of it at this point. Or at least exclude it from commits